py-polars/src/polars/datatypes/classes.py

```diff
@@ -773,44 +832,65 @@ class Categorical(DataType):
 
     Parameters
     ----------
+    categories
+        The categories used for this type; must be a :py:class:`Categories`
+        instance, or a string which is interpreted as the name of a
+        :py:class:`Categories`. If not provided, the global categories
+        (`pl.Categories()`) are used.
+
+        For legacy reasons if the string is either `"physical"` or `"lexical"`,
+        it is ignored and a warning is issued. If you wish to use a `Categories`
+        named `"physical"` or `"lexical"`, please pass it using
+        :py:class:`Categories` explicitly.
+
     ordering : {'lexical', 'physical'}
-        Ordering by order of appearance (`'physical'`, default)
-        or string value (`'lexical'`).
+        This used to specify how this type was ordered, but now does nothing.
 
         .. deprecated:: 1.32.0
             Parameter is now ignored. Always behaves as if `'lexical'` was passed.
+
+    See Also
+    --------
+    Categories
     """
 
     ordering: CategoricalOrdering | None
     categories: Categories
 
     def __init__(
         self,
-        ordering: CategoricalOrdering | Categories | None = "lexical",
-        **kwargs: Any,
+        categories: Categories | str | None = None,
```

Comment:
Isn't this a breaking change?
Unless you make `categories` as a keyword-only argument, I think users will run into problems with existing specifications like `'lexical'`.

Reply:
@eitsupi Can you give an example invocation that worked before but doesn't now?

Reply:
Sorry, I see #25447